pom.xml

```diff
@@ -192,6 +192,64 @@
     <workflow-multibranch.version>2.16</workflow-multibranch.version>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>no-script-splitting</id>
+      <activation>
+        <jdk>[,1.9)</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <systemPropertyVariables>
+                <org.jenkinsci.plugins.pipeline.modeldefinition.parser.RuntimeASTTransformer.SCRIPT_SPLITTING_TRANSFORMATION>false</org.jenkinsci.plugins.pipeline.modeldefinition.parser.RuntimeASTTransformer.SCRIPT_SPLITTING_TRANSFORMATION>
```

Comment:
I do not get it. Why not just set these flags directly in the code, at runtime, depending on the Java version? Or if you want to test in both modes, use `@Parameterized` etc. See https://github.com/jenkinsci/bom/pull/451#issuecomment-796090220